Default bank account definition (#48)
wealthkernel/api-documentation-guides: docs/withdrawals/FAQs.md

There could be a number of reasons a withdrawal has been rejected:

> - The provided bank account or latest to be added bank account is not active.

Reviewer: ```suggestion
- The provided (or the most recently added if no `bankAccountId` was provided) bank account is not active.

```
